src/drt/src/pa/FlexPA_prep.cpp

```diff
@@ -486,81 +527,42 @@ void FlexPA::genAPsFromRect(std::vector<std::unique_ptr<frAccessPoint>>& aps,
   }
 
   // gen all full/half grid coords
-  const int offset = is_macro_cell_pin ? hwidth : 0;
+  const int offset = is_macro_cell_pin && !use_center_line ? hwidth : 0;
```

Comment:
That's not equivalent

Reply:
This comes from the fact that originally `offset` only is used inside the 
```
if (!is_macro_cell_pin || !use_center_line)
```
if statement.
This allows us to reduce offset directly to:
```
offset = is_macro_cell_pin && (!is_macro_cell_pin || !use_center_line) ? hwidth : 0;
```
as the statement already requires ```is_macro_cell_pin``` to be true to even consider the second part it is equivalent to
```
offset = is_macro_cell_pin && !use_center_line ? hwidth : 0;
```

Reply:
I understand, Could you add a comment describing what's happening here so that this doesn't become confusing later.